### PR TITLE
finished the function, need to fix the tests

### PR DESCRIPTION
**Pull request by milicmil**
@rachelkriggs @nazra-inari @jpush1773 

I finished my function. I think the function description may have to be modified in the Readme file. There are three potential discrepancies between pandas data types and what we are describing in the Readme file.

https://pbpython.com/pandas_dtypes.html

- [ ] 1. String and character in pandas is treated as a data type of object. 

- [ ] 2. In a separate case, if the column for example is a mix of integers and dates, the column is defined as an object.

- [ ] 3. In our function description we have an "other" data type, which I am not sure what type it is.

### CODE REVIEW

File: explorePy/variable_summary.py [hunk 1]
```diff
@@ -2,15 +2,57 @@
...
-
     inputs:
-    x: data frame
-
+    frames: data frame
     returns:
     data frame - variable type and count of each type
     """
-    return x.variable_summary()
+    #https://stackoverflow.com/questions/19900202/how-to-determine-whether-a-column-variable-is-numeric-or-not-in-pandas-numpy
```
Comment (rachelkriggs) [inline explorePy/variable_summary.py:14]: we can delete this line (13)

File: explorePy/variable_summary.py [hunk 2]
```diff
@@ -2,15 +2,57 @@
...
+    #pandas data types
```
Comment (rachelkriggs) [inline explorePy/variable_summary.py:15]: we can delete this line (14)

File: explorePy/variable_summary.py [hunk 3]
```diff
@@ -2,15 +2,57 @@
...
+    #https://pbpython.com/pandas_dtypes.html
```
Comment (rachelkriggs) [inline explorePy/variable_summary.py:16]: we can delete this line (15)

File: explorePy/variable_summary.py [hunk 4]
```diff
@@ -2,15 +2,57 @@
...
+
+    int_count = 0
+    char_count = 0
+    bool_count = 0
+    date_count = 0
+    other_count = 0
+
+    for i in range (0, col_len):
+        if frames[cols[i]].dtype == np.int64:
+        #if np.issubdtype(frames[cols[i]].dtype, np.number) == True:
```
Comment (rachelkriggs) [inline explorePy/variable_summary.py:33]: can we delete this line? (31)

File: explorePy/variable_summary.py [hunk 5]
```diff
@@ -2,15 +2,57 @@
...
+        elif frames[cols[i]].dtype == np.object:
+            char_count = char_count + len(frames[cols[i]])
+        elif frames[cols[i]].dtype == np.bool:
+            bool_count = bool_count + len(frames[cols[i]])
+        elif frames[cols[i]].dtype == np.datetime64:
+            date_count = date_count + len(frames[cols[i]])
+        else:
+            other_count = other_count + len(frames[cols[i]])
+    print(char_count)
+
```
Comment (rachelkriggs) [inline explorePy/variable_summary.py:44]: all counters above just simply add 1 if we have identified the correct type

**Review by rachelkriggs — CHANGES_REQUESTED** [review]
Nice work! Please see detailed review comments for changes to be addressed.

File: explorePy/variable_summary.py (hunk 5 above)
Comment (milicmil) [reply explorePy/variable_summary.py:44]: I will fix this in the next pull request

File: explorePy/variable_summary.py (hunk 4 above)
Comment (milicmil) [reply explorePy/variable_summary.py:33]: Whoopsie, 

you are right we do not need it.

File: explorePy/variable_summary.py (hunk 3 above)
Comment (milicmil) [reply explorePy/variable_summary.py:16]: That is true, a print statement will suffice.

File: explorePy/variable_summary.py (hunk 2 above)
Comment (milicmil) [reply explorePy/variable_summary.py:15]: You are right we do not need it.

File: explorePy/variable_summary.py (hunk 1 above)
Comment (milicmil) [reply explorePy/variable_summary.py:14]: No need for that link

**Pull request merged**